pages/Reviews/index.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { hex2bin } from '@shopgate/pwa-common/helpers/data';
+import View from 'Components/View';
+import List from 'Components/Reviews/components/List';
+import LoadMoreButton from './components/LoadMore';
+import connect from './connector';
+import {
```

Comment:
Please make this a single line as it's only importing one thing.

Reply:
done